### ISSUE TITLE
Which package should I use ? 

### ISSUE THREAD

**Issue by juulieen** [ayooby/react-native-calendar-heatmap#21]
Got an error when trying to use this package, here what I did :
- `npm i react-native-calendar-heatmap` 
- Import the component from *react-native-calendar-heatmap* like this : 
```
[...]
import CalendarHeatmap from 'react-native-calendar-heatmap'

[...]
<CalendarHeatmap
  endDate={new Date('2019-03-25')}
  numDays={100}
  colorArray={['#eee', '#D44B79', '#6B1928', '#9F3251', '#360000']}
  values={[{ date: '2016-01-01' }, { date: '2016-01-22' }, { date: '2016-01-30' }]}
/>
[...]
```
as mentioned in the README.md

but got the same error as #17 and #15. However, on the issues #17, the fix was to use an alternative package name from @WrathChaos I did try it :
- `npm i @freakycoder/react-native-calendar-heatmap`
- Import the component from *react-native-calendar-heatmap* like this : 
```
[...]
import CalendarHeatmap from '@freakycoder/react-native-calendar-heatmap'

[...]
<CalendarHeatmap
  endDate={new Date('2019-03-25')}
  numDays={100}
  colorArray={['#eee', '#D44B79', '#6B1928', '#9F3251', '#360000']}
  values={[{ date: '2016-01-01' }, { date: '2016-01-22' }, { date: '2016-01-30' }]}
/>
[...]
```

And it works!

But on the README of this package @freakyCoder/react-native-calendar-heatmap, says to install `react-native-calendar-heatmap` 

I'm a bit lost, can you help me to understand which package should I use?

**Comment by kuraydev** [comment]
My forked branch was just for PR. It's been merged. Let me check what is wrong with the original repo.

**Comment by kuraydev** [comment]
Oh I got it. @ayooby you need to publish the latest version: [0.2.1 ](https://github.com/ayooby/react-native-calendar-heatmap/releases/tag/0.2.1)

**Comment by kuraydev** [comment]
@ayooby Can you add me the npm as a collaborator. I can do it fastly

**Comment by kuraydev** [comment]
@juulieen Until ayooby publish the new version, you can use mine.

**Comment by ayooby** [comment]
Dear @WrathChaos please share with me your NPM username?

**Comment by kuraydev** [comment]
Hi @ayooby,
It's freakycoder.
https://www.npmjs.com/~freakycoder

**Comment by ayooby** [comment]
@WrathChaos I just sent the invitation.
Thank you so much for your help 👍

**Comment by kuraydev** [comment]
> @WrathChaos I just sent the invitation.
> Thank you so much for your help 👍

I tried but I could not find the invitation or even publish the new version :( Can you please try to invite me again? @ayooby

**Comment by ayooby** [comment]
Please check again.
@WrathChaos

**Comment by kuraydev** [comment]
Yes! Thank you @ayooby.

@juulieen I've published the 0.2.1 version. Can you please try it?

**Comment by juulieen** [comment]
Thank a lot, I did try it and it's working great! 😄
